src/main/resources/application.properties

```diff
@@ -1,14 +1,19 @@
 #spring.main.banner-mode=off
 logging.level.org.springframework=ERROR
+logging.level.root=INFO
 
 spring.jpa.show-sql=true
 #Do not use this in production
 spring.jpa.hibernate.ddl-auto=update
+spring.cloud.stream.bindings.input.destination=mySource
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 
 spring.datasource.initialization-mode=always
 spring.datasource.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/${PG_DB_NAME}
```

Comment:
Do you want to add 'saraswati' here? You tell to add saraswati in the instructions but I am not sure where it is used? Test application.properties does have 'sarawati-test'.